src/QsCompiler/Tests.LanguageServer/TestSetup.cs

```diff
@@ -18,8 +18,8 @@ public sealed partial class BasicFunctionality : IDisposable
     {
         // basic setup
 
-        private Connection connection;
-        private JsonRpc rpc;
+        private Connection? connection;
+        private JsonRpc rpc = null!; // Initialized in SetupServerConnectionAsync.
```

Comment:
I can't add a comment in a line that has not been edited but should the null-conditional operator in line 47 be a null-forgiving operator?
`this.rpc?.Dispose();` -> `this.rpc!.Dispose();`

Reply:
No, I don't think so - in that case, if the class is disposed before `this.rpc` is initialized, then `this.rpc.Dispose()` will throw a NullPointerException. I think the intended behavior is to do nothing.